idea/api-gateway/src/common/enums/genesis-hash.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum GenesisRpc {
+    valid = 'genesis.valid'
```

Comment:
`genesis.valid` is strange choice. It doesn't reflect the necessary meaning. Maybe `testBalance.available` will be better

Reply:
And why you use separate enum for this method? I would store it together with the rest of the rpc methods

Reply:
We can create 2 enums, one with all Kafka methods and another one with all api methods.

Reply:
In `@gear-js/common` we already have `API_METHODS` `enum`. Maybe the new `method` add to this `enum`. What do you think @osipov-mit 

https://github.com/gear-tech/gear-js/blob/master/idea/common/src/enums/api-methods.enum.ts#L1

Reply:
In my opinion its better to use separate enums for kafka topics and api methods (and describe all the methods there).